docs/data/migration/migration-pickers-v7/migration-pickers-v7.md

```diff
@@ -256,3 +256,69 @@ const theme = createTheme({
   },
 });
 ```
+
+## Stop passing `utils` and the date/time object to some translation keys
+
+If you are customizing the translation of those keys, you no longer need to provide `utils` and the date/time object to it, but only the formatted date/time value as a string.
+
+When migrating to v8, you must have a small refactor on the usage of those keys:
+
+- `openDatePickerDialogue`:
+
+  ```diff
```

Comment:
Have you considered combining these diffs into one?
I feel like it would create less visual noise. 🤔